src/ontology/mondo-edit.obo

```diff
@@ -584963,6 +584963,20 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8003" xsd:anyURI
 
+[Term]
+id: MONDO:1030009
+name: persistent atrial fibrillation
+def: "A disorder characterized by continuous atrial fibrillation that lasts for more than seven days." [https://orcid.org/0009-0009-6791-365X]
+synonym: "persistent AF" EXACT [PMID:29401239]
+xref: ICD10CM:I48.1 {source="MONDO:equivalentTo"}
+xref: MEDGEN:749907 {source="MONDO:equivalentTo"}
+xref: MESH:D001281 {source="MONDO:relatedTo"}
+xref: SCTID:440059007 {source="MONDO:equivalentTo"}
+xref: UMLS:C2585653 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004981 {source="https://orcid.org/0009-0009-6791-365X"} ! atrial fibrillation
```

Comment:
Please use the PMID as a source. 
Also same question as above about the orcid

Reply:
updated to pmid and removed orcid